src/types.ts

```diff
@@ -257,26 +258,20 @@ export interface IdxMetaMap {
 export interface QueryOpts<K extends IndexKeyType> {
   readonly descending?: boolean;
   readonly limit?: number;
-  includeDocs?: boolean;
```

Comment:
why you remove the includeDocs option?

Reply:
From #315 I've understood that `includeDocs` should be on by default. I can add the option back to exclude docs? That said, I don't understand the use-case, so some context here would be great.

https://github.com/fireproof-storage/fireproof/pull/315#issuecomment-2521246582

Reply:
Talked about this with Chris & Meno (separately) in Discord. Looks like we want the `key` and `value` in the result as well (cc @mabels), so that'll be an adjustment to `allDocs`. 

Regarding `includeDocs`, ideally this would be an option to pass in when wanting to exclude the docs. Hopefully we can do some function overloading here so the types are nicer to work with.

Reply:
yes --- rename it to excludeDocs so that the default is including the docs.